fuseki/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM stain/jena-fuseki:3.6.0
+
+ADD shiro.ini /jena-fuseki/shiro.ini
+
+CMD ["/jena-fuseki/fuseki-server", "--loc=/fuseki", "--update", "/ds"]
```

Comment:
as I don't know anything about fuseki, what would happen if someone used the default shiro.ini file that comes with the base docker image?

Reply:
I was under the impression that the `SPARQLUpdateStore` didn't support authentication, but I must have stumbled across some pretty old [docs](https://rdflib.readthedocs.io/en/2.4.0/_modules/rdflib/plugins/stores/sparqlstore.html#SPARQLUpdateStore). I'll enable pass-through for auth.